src/__Private/LinterCLI.php

```diff
@@ -71,19 +78,16 @@ private function lintFile(
       }
 
       using (new ScopedPerfCounter($class.'#processErrors')) {
-        $all_errors = Vec\concat(
-          $all_errors,
-          $this->processErrors($linter, $config, $errors),
-        );
+        $errorHandler->processErrors($linter, $config, $errors);
```

Comment:
why this approach, rather than keeping as a generator and allowing the traverser to deal with them? Either's fine, just want to understand the decision

Reply:
It's because linters can fix all errors from a file in the same call, so passing a single traversable list of errors won't work. I found this approach cleaner than generating `Traversable<Traversable<Error>>` and using that.